spec/forms/two_factor_setup_form_spec.rb

```diff
@@ -56,26 +57,30 @@
       it 'is valid' do
         user = build_stubbed(:user, phone: valid_phone)
         form = TwoFactorSetupForm.new(user)
-
-        result = {
-          success: true,
-          error: nil,
+        extra = {
           otp_method: 'sms',
         }
+        result = instance_double(FormResponse)
 
+        expect(FormResponse).to receive(:new).
```

Comment:
update spec descriptions? The ones in this file are not *wrong* but they are less specific than the descriptions on the specs for other form objects